guides/common/modules/proc_configuring-keycloak-authentication-with-cac-cards.adoc

```diff
@@ -0,0 +1,19 @@
+[id="configuring-keycloak-authentication-with-cac-cards_{context}"]
+= Configuring {Keycloak} Authentication With CAC Cards
+
+Use this procedure to configure {Project} for {Keycloak} authentication with CAC cards.
+
+.Procedure
+* To authenticate using two-factor authentication, enter the following command:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# hammer auth login oauth \
+--two-factor \
+--oidc-token-endpoint 'https://_{Keycloak-short}.example.com_/auth/realms/ssl-realm/protocol/openid-connect/token' \
+--oidc-authorization-endpoint 'https://_{Keycloak-short}.example.com_/auth' \
+--oidc-client-id '_{foreman-example-com}_-foreman-openidc' \
+--oidc-redirect-uri urn:ietf:wg:oauth:2.0:oob
```

Comment:
This is for configuration usind Hammer

Reply:
Yes, this will work for all three use cases: 
a) Without any auth binding 
b) CAC cards
c) TOTP

Reply:
Yes!

Reply:
And what does this command do? Will a description `To configure OAuth settings, enter the following command` work?